ext/opentelemetry-ext-pymongo/src/opentelemetry/ext/pymongo/__init__.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The opentelemetry-ext-pymongo package allows tracing commands made by the
+pymongo library.
+"""
+
+from pymongo import monitoring
+from opentelemetry.trace import SpanKind, Span
+from opentelemetry.trace.status import Status, StatusCanonicalCode
+
+DATA_BASE_TYPE = "mongodb"
+COMMAND_ATTRIBUTES = ["filter", "sort", "skip", "limit", "pipeline"]
+
+
+def trace_integration(tracer=None):
+    """Integrate with pymongo to trace it using event listener.
+       https://api.mongodb.com/python/current/api/pymongo/monitoring.html
+    """
+    monitoring.register(CommandTracer(tracer))
```

Comment:
If *easily* possible, add some guard against registering the CommandTracer twice.

Reply:
For this we will need to check against private fields in the class, they don't expose any method to retrieve the global listeners, I would prefer to not rely on internal properties that could change anytime

https://github.com/mongodb/mongo-python-driver/blob/master/pymongo/monitoring.py#L465

Reply:
OK, then it's fine to leave this as-is.